package.json

```diff
@@ -36,6 +36,7 @@
     "webpack-dev-server": "^1.14.1"
```

Comment:
Just placing a comment here altough is something more generic: got an error when transpiling the project:

Cannot resolve... students route